### ISSUE TITLE
Release 1.0-beta2; provide "local only" site option; better documentation of the runner API

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/caliper#293]
```
Hello,

Two things:

* 1.0-beta2 is long overdue, I had to git clone and install 1.0-beta-SNAPSHOT 
to have annotations available; when will it be released?
* by default, CaliperMain.main() will upload to microbenchmarks.appspot.com; if 
this is not the desired behaviour you're doomed, because...
* ... there is no documentation whatsoever as to how to change that 
programmatically; even the properties file gives no indication.

I have tried and dug through the source file but its heavy use of modules 
anywhere makes it really hard to follow; and from what I could see the 
OutputFileDumper dumps JSON.

It could be a little better if the source of the webapp at 
microbenchmarks.appspot.com were available but it does not seem to be... In 
essence, you cannot use caliper in a non networked environment!

Not to mention that the existing documentation omits to mention the dependency 
on com.sun.jersey:jersey-core.


```

Original issue reported on code.google.com by `fgalie...@gmail.com` on 10 May 2014 at 6:59

**Comment by blundell** [comment]
Commenting so I get updates
